### PR TITLE
feat: allow logging out via navbar button

### LINKED ISSUE #324: Log out
As a user I want to be able to log out from the bookkeeper portal.
Therefore a sign out button should be placed in the nav bar. Clicking that button performs a log out.
The user should be informed, that the log out was effective.

image
image

### Dependencies

none

### Acceptance Criteria

- [ ]  Nav bar contains log out button (icon bi-box-arrow-left)
- [ ]  Notification, user and log out button have a size of **fs-4** und a padding of 3.
- [ ] Clicking the log out button performs a log out. The log in screen is opened and an info box informs the user: "You have been successfully logged out"

### PR DESCRIPTION
**Pull request by Toalaah**
closes #324

### CODE REVIEW

File: app/views/layouts/_navbar.html.erb [hunk 1]
```diff
@@ -57,6 +57,15 @@
...
 
+        <% if user_signed_in? %>
+          <li class="nav-item">
+            <%= link_to destroy_user_session_path, data: { turbo_method: :delete }, class: "nav-link" do %>
+              <i class="bi bi-box-arrow-left mobile-only"></i>
+              <%= t("navbar.sign_out") %>
+            <% end %>
+          </li>
+        <% end %>
+
```
Comment (yannik-dittmar) [inline app/views/layouts/_navbar.html.erb:68]: I searched for this button for ages in the desktop view 😅
I thought you had implemented it as the mock-up shows (a log-out btn in the right navbar side) and wondered if I had done something wrong with my setup.
I think we should do it the way the mock-up shows. At least that is my personal preference.
What do you think?

**Comment by yannik-dittmar** [comment]
And before I give the go for merging could you maybe cleanup your commit history?
Instead of updating the branch with a merge commit every time you can rebase it.
This prevents from these ugly merger commits piling up.

File: app/views/layouts/_navbar.html.erb (hunk 1 above)
Comment (Toalaah) [reply app/views/layouts/_navbar.html.erb:68]: Yeah this was admittedly a little sloppy. I have gone ahead and adapted the navbar to more closely match the mockup in 6b7c33e3a73b6a1b536b62b5582518185a9a6f70.

**Review by yannik-dittmar — APPROVED** [review]
LGTM

**Pull request merged**